pyroll/core/roll_pass/hookimpls/roll_pass.py

```diff
@@ -94,6 +96,11 @@ def rel_elongation(self: RollPass):
     return self.abs_elongation / self.in_profile.length
 
 
+@RollPass.strain
+def strain(self: RollPass):
+    return self.log_elongation
+
+
 @RollPass.strain_rate
 def strain_rate(self: RollPass):
```

Comment:
Should be altered to plugin since this is Hoff and Dahl.

Reply:
Correct. I think we should sit together and go through every core hookimpl and decide.